File: JS/Factorial_Iterative_No_Input.js
n = 1



function factorial(n) {
    var factorial_total = 1;
    for (iterator = 1; iterator < n; iterator++) {
        factorial_total = factorial_total * (iterator + 1);
    }
    return factorial_total
}


for (n = 1; n < 10; n++){
    total = factorial(n)
    console.log("The factorial of ", n, " is ", total, ".\n")
}

File: python/Factorial_iterative_no_input.py

print("This program finds the factorial of a number.\n")

n = 0

def factorial(n):
    factorial_total = 1
    iterator = 1
    while iterator < n:
        iterator += 1
        factorial_total = factorial_total * iterator
    return factorial_total

def n_iterator(n):
    while n < 10:
        n += 1
        factorial_total = factorial(n)
        print("The factorial of ", n, " is ", factorial_total)

n_iterator(n)
